sql.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+language: SQL
+filename: learnsql.sql
+contributors:
+- ["Bob DuCharme", "http://bobdc.com/"]
+lang: en-en
```

Comment:
Please remove this line

```
lang: en-en
```

This is not necessary for English language docs and creates an issue where they are not added to the site build.

Reply:
I made the change and resolved it, but I think that it broke the build according to https://travis-ci.org/adambard/learnxinyminutes-docs/builds/448955911?utm_source=github_status&utm_medium=notification .